### PR TITLE
Master add hostname web UI

### PR DESCRIPTION
**Pull request by jkonecny12**
Add hostname implementation to the web UI.

Missing:
- [ ] Add tests

### CODE REVIEW

File: ui/webui/src/network/Hostname.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    const onApplyClicked = () => {
+        hostnameProxy.SetCurrentHostname(hostname);
+    };
+
+    return (
+        <Flex>
+            <FlexItem>
+                <TextInput
+                  value={hostname}
+                  type='text'
```
Comment (KKoukiou) [inline ui/webui/src/network/Hostname.jsx]: no need for this. It;'s the default

File: ui/webui/src/network/Hostname.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                  onhhange={setHostname}
+                  arha-label='hostname text input' />
+            </FlexItem>
+            <FlexItem><Button variant='secondary' onClick={onApplyClicked}>Apply now</Button></FlexItem>
```
Comment (KKoukiou) [inline ui/webui/src/network/Hostname.jsx:60]: I find it weird, to have both done and apply now. I would skip this :)

File: ui/webui/src/network/Hostname.jsx (hunk 1 above)
Comment (jkonecny12) [reply ui/webui/src/network/Hostname.jsx]: Fixed.

File: ui/webui/src/network/Hostname.jsx (hunk 2 above)
Comment (jkonecny12) [reply ui/webui/src/network/Hostname.jsx:60]: Design decision we have to make. It's a separate commit so it should be fine.

File: ui/webui/src/network/Hostname.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    const onApplyClicked = () => {
+        hostnameProxy.SetCurrentHostname(hostname);
+    };
+
+    // FIXME: Add validation of the hostname
+    return (
+        <Flex>
+            <FlexItem>
+                <TextInput
```
Comment (KKoukiou) [inline ui/webui/src/network/Hostname.jsx:55]: I think I would encapsulate this inside a [Form](https://www.patternfly.org/v4/components/form) and create a label -> item pair as seen in this example. Tight now it's not clear at all what this refers to.

File: ui/webui/src/network/Hostname.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                  value={hostname}
+                  onChange={setHostname}
+                  aria-label='hostname text input' />
```
Comment (KKoukiou) [inline ui/webui/src/network/Hostname.jsx:58]: The aria-label is needed when there is not a label. If you follow my suggestion from above you can remove this line.

**Review by KKoukiou — COMMENTED** [review]
It's close :) and just missing tests now.

**Comment by jkonecny12** [comment]
Closing in favor of: https://github.com/rhinstaller/anaconda/pull/3788

**Pull request closed without merging**